testsuite/tests/test/crate-init/test.py

```diff
@@ -0,0 +1,16 @@
+"""
+Test the skeleton tests crate created in `alr init`
+"""
+
+import os.path
+from drivers.alr import run_alr
+from drivers.asserts import assert_match
+
+run_alr("init", "--lib", "xxx")
+os.chdir("xxx")
+
+p = run_alr("test", complain_on_error=False)
+assert_match(".*\[ FAIL \] tests-example_test.*", p.out)
+# default test after init always fails
```

Comment:
Do we really want that all new crates come with a failing test? This is a chore we'll be imposing on all of our users.

Reply:
I didn't want users to inadvertently commit the 'example_test' file, so that's why I made it fail by default. I considered not putting an example test at all, although I like the self-documenting effect...

Reply:
I think I will just change this to PASS, so it's less of a pain point. Unless we want to go all the way to add annotations to test files, like a magic `--  XFAIL` comment on the first line, or a filename suffix.